### PR TITLE
Added some ML Books 

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
Added some great free ML Books

## What does this PR do?
Add Resource(s) 

### Description 

* [Machine Learning from Scratch](https://dafriedman97.github.io/mlbook/content/introduction.html): Free online book "Machine Learning from Scratch" 
* [Understanding Machine Learning: From Theory to Algorithms](https://www.cs.huji.ac.il/~shais/UnderstandingMachineLearning/understanding-machine-learning-theory-algorithms.pdf)

### Why is this valuable (or not)
Both are great collections containing in-depth tutorials with a brief explanation of mathematical prerequisites.

### How do we know it's really free?
Both are available online 
### For book lists, is it a book?
Yes, it is a book
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by eshellman** [comment]
Please read CONTRIBUTING.md guidelines!
be sure to check the output of Travis-CI for linter errors

**Comment by SauravMaheshkar** [comment]
**Linter-errors** fixed 👍

File: free-courses-en.md [hunk 1]
```diff
@@ -301,11 +301,13 @@
 
 * [Google's Machine Learning Crash Course](https://developers.google.com/machine-learning/crash-course/)
 * [Intro to Machine Learning Udacity](https://www.udacity.com/course/intro-to-machine-learning--ud120) (Udacity)
+* [Machine Learning from Scratch](https://dafriedman97.github.io/mlbook/content/introduction.html)
```
Comment (eshellman) [inline free-courses-en.md:304]: duplicate

File: free-courses-en.md [hunk 2]
```diff
@@ -301,11 +301,13 @@
...
 * [Machine Learning Mini Bootcamp](https://lambdaschool.com/free-course-machine-learning/)
 * [Pattern Recognition and Machine Learning](https://www.microsoft.com/en-us/research/people/cmbishop/#!prml-book)
 * [Principles of Machine Learning By Microsoft](https://www.edx.org/course/principles-machine-learning-microsoft-dat203-2x-6)
 * [PyTorch tutorials by PyTorch.org](https://pytorch.org/tutorials)
 * [Stanford University Machine Learning](https://www.coursera.org/learn/machine-learning)
+* [Understanding Machine Learning: From Theory to Algorithms](https://www.cs.huji.ac.il/~shais/UnderstandingMachineLearning/understanding-machine-learning-theory-algorithms.pdf) (PDF)
```
Comment (eshellman) [inline free-courses-en.md:310]: duplicate

**Pull request closed without merging**